src/js/va-letters/containers/VALettersApp.jsx

```diff
@@ -1,19 +1,60 @@
+import PropTypes from 'prop-types';
 import React from 'react';
 import { connect } from 'react-redux';
 
+import RequiredLoginView from '../../common/components/RequiredLoginView';
+
+// This needs to be a React component for RequiredLoginView to pass down
+// the isDataAvailable prop, which is only passed on failure.
+function AppContent({ children, isDataAvailable }) {
+  const unregistered = isDataAvailable === false;
+  let view;
+
+  if (unregistered) {
+    view = (
+      <h4>
+        To view and download your VA letters, you need something something something.
+      </h4>
+    );
+  } else {
+    view = children;
+  }
+
+  return (
+    // TODO: add HTML and CSS when design has settled
+    <div className="row">
+      {view}
+    </div>
+  );
+}
+
 class VALettersApp extends React.Component {
   render() {
-    const { children } = this.props;
     return (
-      <div>{children}</div>
+      <RequiredLoginView
+          authRequired={3}
+          serviceRequired={"evss-letters"}
```

Comment:
It can be anything--was just following suit with `evss-claims` but right now it's just a placeholder for whatever vets-api decides to name this service, to be defined in https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/backend_services.rb

I just switched it to pretend to be `user-profile` since we're not going to be writing any submission code for a while and right now just need to test that the auth stuff works as intended.